nxviz/encodings.py

```diff
@@ -103,15 +103,15 @@ def color_func(data: pd.Series) -> Callable:
     return partial(func, cmap=cmap, data=data)
 
 
-def data_color(data: pd.Series) -> pd.Series:
+def data_color(data: pd.Series, ref_data: pd.Series) -> pd.Series:
     """Return iterable of colors for a given data.
 
     `cfunc` gives users the ability to customize the color mapping of a node.
     The only thing that we expect is that it takes in a value
     and returns a matplotlib-compatible RGB(A) tuple or hexadecimal value.
```

Comment:
Suggesting that we explain `ref_data` a bit more clearly.

```suggestion
    `cfunc` gives users the ability to customize the color mapping of a node.
    The only thing that we expect is that it takes in a value
    and returns a matplotlib-compatible RGB(A) tuple or hexadecimal value.

    The function takes in `ref_data`
    which is used to determine important colormap values (such as boundaries).
    That colormap is then applied to the actual `data`.

    :param data: The data on which to map colors.
    :param ref_data: The data on which the colormap is constructed.
```

Reply:
👍